src/UI/Calendar/CalendarFeedView.js

```diff
@@ -18,6 +19,7 @@ module.exports = Marionette.Layout.extend({
     events : {
         'click .x-includeUnmonitored' : '_updateUrl',
         'click .x-premiersOnly'       : '_updateUrl',
+        'click .x-allDay'             : '_updateUrl',
         'itemAdded .x-tags'           : '_updateUrl',
         'itemRemoved .x-tags'         : '_updateUrl'
     },
```

Comment:
While you're at it can you replace 'NzbDrone' with 'Sonarr' a few lines down, the backend already supports it and evidently we forgot to replace it in the UI.